test/streams/broadcastable_test.rb

```diff
@@ -224,6 +224,18 @@ class Turbo::BroadcastableTest < ActionCable::Channel::TestCase
       @message.broadcast_render_to @profile
     end
   end
+
+  test "broadcasting morph to stream now" do
+    assert_broadcast_on "stream", turbo_stream_action_tag("morph", target: "message_1", template: render(@message)) do
+      @message.broadcast_morph_to "stream", target: "message_1"
+    end
+  end
+  
+  test "broadcasting morph now" do
```

Comment:
Maybe add a test with `children-only` to make sure the option works?

Reply:
Added